distribution/test/itests/test-itests-common/src/test/java/ddf/common/test/ServiceManager.java

```diff
@@ -74,11 +74,11 @@
 
     private static final int CONFIG_UPDATE_WAIT_INTERVAL_MILLIS = 5;
 
-    public static final long MANAGED_SERVICE_TIMEOUT = TimeUnit.MINUTES.toMillis(10);
+    public static final long MANAGED_SERVICE_TIMEOUT = TimeUnit.MINUTES.toMillis(100);
```

Comment:
That's 100 minutes. `toMillis` does the time unit math conversion.

Instead of converting your `TimeUnit`s to millis at all, could you just use them directly?